examples/requirements.txt

```diff
@@ -0,0 +1,50 @@
+amqp==1.4.9
```

Comment:
It happened. I'm not sure if with `>=X.Y` you meant to avoid specifying the patch level, but in any case this is currently impossible:
```shell
% pip install "invenio-db>=1.0"
Collecting invenio-db>=1.0
  Could not find a version that satisfies the requirement invenio-db>=1.0 (from versions: 1.0.0a1, 1.0.0a2, 1.0.0a3, 1.0.0a4, 1.0.0a5, 1.0.0a6, 1.0.0a7, 1.0.0a8, 1.0.0a9, 1.0.0a10)
No matching distribution found for invenio-db>=1.0
```